task8/index.js

```diff
@@ -0,0 +1,49 @@
+const obj = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+const div = document.getElementById("js-parent");
+
+const loading = () => {
+    const loadingPlace = document.createElement("div");
+    const gif = document.createElement('img');
+    loadingPlace.id = "loadingPlace";
+    gif.src = "loading-circle.gif";
+    div.appendChild(loadingPlace).appendChild(gif);
+};
+
+const loaded = () => {
+    const loadingPlace = document.getElementById("loadingPlace");
+    loadingPlace.remove();
+}
+    
+const getData  = new Promise((resolve, reject) => {
+    loading();
+        setTimeout(() => {
+        reject(obj);
+    },3000);
```

Comment:
Although obj is passed as an argument to reject, it is not used in .catch.

So
reject is for when the problem cannot be solved, so here is what I would do.

IMO
```suggestion
        setTimeout(() => {
        reject(”error!”);
    },3000);
```

Reply:
I am sorry that I do not understand the intent of the problem.
With what I wrote, it makes no sense to use .catch. 
Thanks again!